hal/halx86/acpi/madt.c

```diff
@@ -192,41 +162,30 @@ HalpParseApicTables(
 
                 if (AcpiHeader->Length != sizeof(*InterruptOverride))
                 {
-                    DPRINT01("Type/Length mismatch: %p, %u\n", AcpiHeader, AcpiHeader->Length);
+                    DPRINT("Type/Length mismatch: %p, %u\n", AcpiHeader, AcpiHeader->Length);
                     return;
                 }
 
-                DPRINT00(" Interrupt Override: Bus %u, SourceIrq %u, GlobalIrq %08X, IntiFlags %04X / UNIMPLEMENTED\n",
+                /*
+                 * We use this information to make sure IOAPIC knows the difference between the
+                 * legacy PIC IRQs and the associated GSI with them.
+                 */
+                DPRINT1(" Interrupt Override: Bus %u, SourceIrq %u, GlobalIrq %08X, IntiFlags %04X / UNIMPLEMENTED\n",
```

Comment:
Would `DPRINT(...);` (as intended) + (temporary and separated) `UNIMPLEMENTED;` work?

Reply:
I'm going to end up implementing this here soon, so I'd just leave it as is.